docs/contributing/lint.md

```diff
@@ -29,15 +29,15 @@ make pre-commit-install
 Now pre-commit (and so configured hooks) will run automatically on `git commit` on each changed file.
 However it is also possible to trigger it against all files.
 
-- Note: Hadolint pre-commit uses docker to run, so docker should be running while running this command.
+_Note: Hadolint pre-commit uses docker to run, so docker should be running while running this command._
```

Comment:
Since we use MyST markdown parsing, I think we can do this instead for a visual helping to make this "look like a note."

````suggestion
```{note}
Hadolint pre-commit uses docker to run, so docker should be running while running this command.
```
````

Reply:
Yes, thank you!